trade-finance-manager-ui/templates/case/parties/_macros/parties-indemnifier-area.njk

```diff
@@ -102,10 +102,7 @@
             key: 'Approval level required',
             value: ''
           }) }}
-        {% if not userCanEdit %}
```

Comment:
Does it look fine if the window is made smaller?

Reply:
Yep